virttest/utils_test/__init__.py

```diff
@@ -1601,7 +1601,10 @@ def config_control(control_path, job_args=None):
     except Exception:
         # Leak global_config.ini, generate a mini configuration
         # to ensure client tests can work.
-        import ConfigParser
+        try:
+            import configparser as ConfigParser
+        except ImportError:
```

Comment:
I think python2 support is deprecated in avocado-vt. Only keep the python3 is enough.
@chunfuwen Could you help to confirm?

Reply:
That's right

Reply:
@xiaodwan @chunfuwen @chloerh 
Thanks for the review. I am very sorry for the delay in my response. I didn't notice the review on this PR. some how I missed it.

With changes in this PR, we are using only the python3 ones right? Just to avoid changes in each and every place throughout the file, I am importing the py3 one as py2. so that every where it can be py2. like creating an alias for py3 as py2.

Are you suggesting me to change every where in the code and use only the py3 ones?

Reply:
Yes, avocado-vt only supports python3.

Reply:
@xiaodwan even the changes in this PR deals with python3 only. It is just that importing py3 library as py2 one just to avoid avoid the changes through out the file.  Do you want me to change every where in the file? ?

Reply:
Yes. But I'd like to hear what @chunfuwen and @chloerh  say. 
IMO, we should give up all python2 support even there are still some codes like this example in some other files.

Reply:
@chunfuwen @chloerh 
Could you please provide your valuable comments here.

Shall I change this PR to have changes through out the file to have only py3 library names?
If yes, I will change all other required files too.

Reply:
Thaks @chloerh.
As per the comments received, I have removed all the py 2 references in the file. If this is what was expected, I will change all other files accordingly wherever necessary.

@chloerh @xiaodwan @chunfuwen Please take a look at the recent changes.